### PR TITLE
Feature/single image

### PR DESCRIPTION
**Pull request by rpoluri**
### :pencil: Description
Add allow-grant.sh to metastore docker container, so that we can use same image for init container also.

### CODE REVIEW

**Comment by abhimanyugupta07** [comment]
So, it looks like we won't require this https://github.com/ExpediaGroup/apiary-metastore-docker/tree/master/initContainer. Will this then be removed in another PR ?

**Comment by rpoluri** [comment]
Yes, we shouldn't need separate container, will create another PR to cleanup that.

**Review by abhimanyugupta07 — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -79,4 +79,5 @@ COPY files/update_property.py /bin/update_property.py
 COPY files/s3inventory.tpl /s3inventory.tpl
 COPY files/startup.sh /startup.sh
 COPY files/s3_inventory_repair.sh /s3_inventory_repair.sh
+COPY files/allow-grant.sh /allow-grant.sh
```
Comment (KenFigueiredo) [inline Dockerfile:82]: Do we still need `db-iam-user.sh`?

**Comment by KenFigueiredo** [comment]
@rpoluri @abhimanyugupta07 
Updated to remove the secondary container in favor a single container approach:
https://github.com/ExpediaGroup/apiary-metastore-docker/pull/84

**Pull request merged**